opensplice_cmake_module/env_hook/opensplice.bat.in

```diff
@@ -1,4 +1,27 @@
-set "OSPL_HOME=@OpenSplice_HOME@"
+set "_DEFAULT_OSPL_HOME=@OpenSplice_HOME@/"
```

Comment:
I would suggest to call this variable `_BUILTTIME_OSPL_HOME` (or similar) to clarify the semantic. `default` doesn't make much sense imo.